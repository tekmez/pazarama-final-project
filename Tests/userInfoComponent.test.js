import React from "react";
import renderer from "react-test-renderer";
import Info from "../src/components/UserInfo/index";
test("User Info Component", () => {
  const tree = renderer
    .create(
      <Info user={{firstName: "Deneme", lastName: "Test", age: "25", reason: "reason", address: "address", status: "status", answer: "answer", tc: "IDn"}} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});