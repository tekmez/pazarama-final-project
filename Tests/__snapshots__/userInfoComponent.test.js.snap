// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Info Component 1`] = `
<div
  className="user-info"
>
  <h2
    className="title"
  >
    Name:
     
    <span
      className="info"
    >
      Deneme
       
      Test
    </span>
  </h2>
  <h2
    className="title"
  >
    Age: 
    <span
      className="info"
    >
      25
    </span>
  </h2>
  <h2
    className="title"
  >
    IDN:
    <span
      className="info"
    >
      IDn
    </span>
  </h2>
  <h2
    className="title"
  >
    Reason: 
    <span
      className="info"
    >
      reason
    </span>
  </h2>
  <h2
    className="title"
  >
    Address: 
    <span
      className="info"
    >
      address
    </span>
  </h2>
  <h2
    className="title"
  >
    Status: 
    <span
      className="info"
    >
      status
    </span>
  </h2>
  <h2
    className="title"
  >
    Answer: 
    <span
      className="info"
    >
      answer
    </span>
  </h2>
</div>
`;
